src/org/zaproxy/zap/extension/advreport/ExtensionAdvReport.java

```diff
@@ -38,6 +40,8 @@
 public class ExtensionAdvReport extends ExtensionAdaptor {
 
 	public static final String NAME = "ExtensionAdvReport";
+	//Alert types with corresponding risk Levels (High, Medium, Low, Informational)
+	public final Map<String, String> AlertTypeRisk = new HashMap<>();

```

Comment:
- Maybe it's enough with default access?
 - I'd prefer not seeing this as instance variable, the information is not needed once the dialogue is dismissed, but since it requires a couple of changes and it's still alpha I think it can be improved later.